core/templates/pages/exploration-editor-page/history-tab/history-tab.directive.ts

```diff
@@ -238,26 +240,22 @@ angular.module('oppia').directive('historyTab', [
                 }
               },
               controller: [
-                '$scope', '$uibModalInstance', 'version',
+                '$controller', '$scope', '$uibModalInstance', 'version',
                 'ExplorationDataService',
                 function(
-                    $scope, $uibModalInstance, version,
+                    $controller, $scope, $uibModalInstance, version,
                     ExplorationDataService) {
-                  $scope.version = version;
+                  $controller('ConfirmOrCancelModalController', {
+                    $scope: $scope,
+                    $uibModalInstance: $uibModalInstance
+                  });
 
+                  $scope.version = version;
                   $scope.getExplorationUrl = function(version) {
                     return (
                       '/explore/' + ExplorationDataService.explorationId +
                       '?v=' + version);
                   };
-
-                  $scope.revert = function() {
-                    $uibModalInstance.close(version);
-                  };
-
-                  $scope.cancel = function() {
-                    $uibModalInstance.dismiss('cancel');
-                  };
                 }
               ]
             }).result.then(function(version) {
```

Comment:
Do we need to handle the cancel case here ?

Reply:
This is for confirm case, we passing the version through the template.

Reply:
I mean for the case where we don't handle the dismiss callback we see console error, can it happen here? (I don't see a callback function for the disimss.)